internal/db/utils.go

```diff
@@ -24,3 +35,20 @@ func IsValidSqldUrl(uri string) (bool, string) {
 func EscapeSingleQuotes(value string) string {
 	return strings.Replace(value, "'", "''", -1)
 }
+
+func HasSpecialCharacter(value string) bool {
+	if StartsWithNumber(value) {
+		return true
+	}
+	for _, char := range specialCharacters {
```

Comment:
can't we transform the string into a Set, so we don't have to check for duplicates?